README.md

```diff
@@ -141,6 +142,24 @@ documentation](https://github.com/open-telemetry/opentelemetry-collector/blob/ma
 Additional troubleshooting information can be found in the [getting
 started](getting-started) READMEs.
 
+## Migrating from the SignalFx Smart Agent
```

Comment:
Thoughts on moving this into docs folder? Say `docs/signalfx-smart-agent-migration.md`?

Reply:
I think that's a better location for the migration guide (moved).  Ok to keep this introduction here though?